nxdrive/engine/dao/sqlite.py

```diff
@@ -1610,7 +1610,11 @@ def update_remote_parent_path_dt(
         """
         Used in Direct Transfer to update remote_parent_path and remote_state of a folder's children.
         """
+        if not self.queue_manager:
+            return
+
         local_parent_path = str(local_parent_path)
```

Comment:
Could you do the `str()` thing when calling that method and get rid of that line?